db/migrations/20221031120849_orders.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE orders (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    check_in_date DATETIME NOT NULL,
+    check_out_date DATETIME NOT NULL,
+    user_id INT NOT NULL,
+    room_id INT NOT NULL,
+    order_status_id INT NOT NULL,
```

Comment:
최초 예약 시 예약 대기 혹은 결제 대기 상태를 기본값으로 갖고 있으면 좋을 것 같습니다!
```suggestion
    order_status_id INT NOT NULL DEFAULT 1,
```